policyengine_us/parameters/gov/states/ar/tax/income/credits/personal/aged/base.yaml

```diff
@@ -0,0 +1,17 @@
+description: Arkansas provides this personal tax credit amount for aged filers.
+values:
+  2021-01-01: 29
+
+metadata:
+  unit: currency-USD
+  period: year
+  label: Arkansas aged personal tax credit
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions, LINE 7A.
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=12
+    - title: 2022 AR1000F ARKANSAS INDIVIDUAL INCOME TAX RETURN Full Year Resident
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf#page=1
+    - title: Ark. Code § 26-51-501 (a)(1)(D)
```

Comment:
the amount does not align with the legal code?

Reply:
I guess the legal code is for latest year 2023, the values used are from tax form 2022

Reply:
$20 personal credit for year 2004, changed to $21 in year 2005